cli/src/test/integration-tests/verify-longpaths.sh

```diff
@@ -0,0 +1,9 @@
+if doIsWindows
+then
+  echo "Running ensure git config longpaths gets set integration test"
+  touch "$HOME"/.gitconfig
```

Comment:
Without this line, you will reveal if your code can also handle the case to create the file from scratch properly.
You must not merge/accept this. If you leave it like this, you will also test that the INI parsing can handle empty files...
```suggestion
```